analyzer/modules/consensusaccounts.go

```diff
@@ -17,14 +17,14 @@ const (
 // ConsensusAccountsHandler implements support for transforming and inserting data from the
 // `consensus_accounts` module for a runtime into target storage.
 type ConsensusAccountsHandler struct {
-	source storage.RuntimeSourceStorage
-	qf     *analyzer.QueryFactory
-	logger *log.Logger
+	source      storage.RuntimeSourceStorage
+	runtimeName string
```

Comment:
I think it would make sense to use the `analyzer.api.Runtime` type for runtimes everywhere.

Reply:
Yes, that was sloppy; I used the minimally-invasive, minimal-thought-required option (using a string directly) when I was in the middle of the big refactor. Should have cleaned up later. Thanks for calling it out, fixed now.